docs/rec-description.md

```diff
@@ -0,0 +1,11 @@
+# Brief of the recommendation doc, description of api response
+
+The api doc is [here](https://docs.chaordic.com.br/v0-api-vitrines/docs/get-pages-recommendations).
+
+## API usage
+
+- Insert valid inputs, on this repo is used the [Onsite SDK](https://github.com/chaordic/engage-onsite-sdk-js) for validate and make the api request.
+- Render the widget based on the response.
+- Listen impression (track if the widget is in the user ViewPort).
```

Comment:
Não sei o quão importante seria avisar aqui de alguma maneira.
Mas é possível configurar na clients para a impressão (tanto HIDDEN como VIEWABLE) ser totalmente gerenciada automaticamente pelo backend (onsite-server).

Reply:
Acredito que `Listen` deve ser acompanhado por `to` ou `for`.

A não ser que você queira chamar a atenção de alguém (Listen!) igual a fadinha do Zelda Ocarina of time.